cognite/client/data_classes/filters.py

```diff
@@ -608,9 +677,17 @@ class ContainsAll(FilterWithPropertyAndValueList):
     Example:
 
         Filter than can be used to retrieve items where the property value contains both 42 and 43:
+        Generic example:
 
             >>> from cognite.client.data_classes.filters import ContainsAll
-            >>> filter = ContainsAll(("some", "property"), [42, 43])
+            >>> filter = ContainsAll(('space', 'view_external_id/view_version', 'some_property'), [42, 43])
+
+        Using property on View:
```

Comment:
"Composing a property reference using the `View:as_property_ref` method." would be more descriptive. This applies to all examples